.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "src/github.com/cloudfoundry/bosh-softlayer-cpi"]
 	path = src/github.com/cloudfoundry/bosh-softlayer-cpi
-	url = https://github.com/cloudfoundry/bosh-softlayer-cpi
+	url = https://github.com/petergtz/bosh-softlayer-cpi
```

Comment:
True. I've removed the commit from this PR that changes the .gitmodules. It's pointing to https://github.com/cloudfoundry/bosh-softlayer-cpi again.